product_docs/docs/pgd/5/reference/conflicts.mdx

```diff
@@ -17,39 +17,41 @@ PGD recognizes the following conflict types, which can be used as the `conflict_
+## Conflict resolution
 
-Most conflicts can be resolved automatically. PGD defaults to a last-update-wins mechanism or, more accurately, the `update_if_newer` conflict resolver. This mechanism retains the most recently inserted or changed row of the two conflicting ones based on the same commit timestamps used for conflict detection. The behavior in certain corner-case scenarios depends on the settings used for `bdr.create_node_group` and alternatively for `bdr.alter_node_group`.
+Most conflicts can be resolved automatically. PGD defaults to a last-update-wins mechanism or, more accurately, the `update_if_newer` conflict resolver. This mechanism retains the most recently inserted or changed row of the two conflicting ones based on the same commit timestamps used for conflict detection. The behavior in certain corner-case scenarios depends on the settings used for `bdr.create_node_group` and, alternatively, for `bdr.alter_node_group`.
 
 PGD lets you override the default behavior of conflict resolution by using the following function.
-
-## List of conflict resolvers
+<!-- There is no function that follows >
+### List of conflict resolvers
 
 Several conflict resolvers are available in PGD, with differing coverages of the conflict types they can handle:
 
 -   `error` &mdash; Throws error and stops replication. Can be used for any conflict type.
 -   `skip` &mdash; Skips processing the remote change and continues replication with the next change. Can be used for `insert_exists`, `update_differing`, `update_origin_change`, `update_missing`, `update_recently_deleted`, `update_pkey_exists`, `delete_recently_updated`, `delete_missing`, `target_table_missing`, `target_column_missing`, and `source_column_missing` conflict types.
 -   `skip_if_recently_dropped` &mdash; Skips the remote change if it's for a table that doesn't exist downstream because it was recently (within one day) dropped on the downstream. Throw an error otherwise. Can be used for the `target_table_missing` conflict type. `skip_if_recently_dropped` conflict
-resolver can pose challenges if a table with the same name is re-created shortly after it's dropped. In that case, one of the nodes might see the DMLs on the re-created table before it sees the DDL to re-create the table. It then incorrectly skips the remote data, assuming that the table is recently dropped, and causes data loss. We hence recommend that you don't reuse the object names immediately after they're dropped along with this conflict resolver.
+resolver can pose challenges if a table with the same name is re-created shortly after it's dropped. In that case, one of the nodes might see the DMLs on the re-created table before it sees the DDL to re-create the table. It then incorrectly skips the remote data, assuming that the table is recently dropped, and causes data loss. We therefore recommend that, when using this conflict resolver, you don't reuse the object names immediately after they're dropped.
```

Comment:
I had a hard time following the last sentence. Is this edit correct?

Reply:
Looks fine.